thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -138,7 +138,7 @@
       </div>
 
       {{#if context.urns.size}}
-        <div class="card-container__header">
+        <div class="card-container__header rootcause-tabs">
```

Comment:
undocumented change? what does this do? (nit obvsly)

Reply:
I needed to check whether tabs are absent in the acceptance test, and I needed a unique class name to identify the tabs div. I figured the test will be more readable with `find('.rootcause-tabs').get(0)` as opposed to something like `find('.card-container__header').get(3)`.